container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/user_services_functions/persistent_directories.go

```diff
@@ -96,16 +93,7 @@ func preparePersistentDirectoriesResources(
 
 		persistentVolumeSize := int64(persistentDirectory.Size)
 
-		var persistentVolume *apiv1.PersistentVolume
-		if persistentVolume, err = kubernetesManager.GetPersistentVolume(ctx, volumeName); err != nil {
-			persistentVolume, err = kubernetesManager.CreatePersistentVolume(ctx, namespace, volumeName, volumeLabelsStrs, persistentVolumeSize)
-			if err != nil {
-				return nil, stacktrace.Propagate(err, "An error occurred creating the persistent volume for '%s'", persistentDirectory.PersistentKey)
-			}
-			volumesCreated[persistentVolume.Name] = persistentVolume
-		}
-
-		// For now, we have a 1:1 mapping between volume and volume claims, so it's fine giving it the same name
+		// This claim works with a dynamic driver - it will spin up its own volume - the volume will get deleted when said claims is deleted
```

Comment:
just out of curiosity, Did you test this with Minikube? Does Minikube create a hostPath volume?

Reply:
Minikube uses `standard` not `local-path` I have fixed the doc I mistakenly editted. It should support dynamic provisioning https://platform9.com/blog/tutorial-dynamic-provisioning-of-persistent-storage-in-kubernetes-with-minikube/